docs/getting-started.md

```diff
@@ -58,14 +58,55 @@ installed:
 esy x Hello
 ```
 
+### Where is the binary executable?
+When a compiled binary is run with `esy x Hello`, esy creates a local install sandbox with `/bin`, `/lib`, `/etc` and other directories found globally where binaries are meant to be installed. If you're curious, you could peek into them, running
+
+```shell
+esy echo #{self.install}
+```
+
+and inspect the contents yourself. You'll find the binaries in the `bin` directory.
+
+However, they are not meant to be run directly as they could be missing the necessary [exported environment](./concepts.md) - it could be possible that the binary you created needs a dependency during the runtime. `esy x <your project binary>` is the recommended way to run them.
+
+Checkout [concepts](./concepts.md) for more information.
+
 ## Rebuild the project
 
 Hack on project's source code and rebuild the project:
 
+```shell
+esy 
+```
+
+This will take care of updating dependencies as well as building the project. Same as running `esy install` and `esy build` sequentially.
+
+If you're sure, dependencies haven't been updated, simply run `esy build`
```

Comment:
I think we can remove this suggestion, `esy install` is really fast for no-op and if `esy install` is a no-op (dependencies has changed) then `esy build` will fail. So it's always better just to run `esy`.

Reply:
Right!